ui/server/src/main/scala/pl/touk/nussknacker/ui/process/ProcessCategoryService.scala

```diff
@@ -0,0 +1,32 @@
+package pl.touk.nussknacker.ui.process
+
+import java.util.Collections
+
+import com.typesafe.config.{Config, ConfigFactory}
+import pl.touk.nussknacker.engine.ProcessingTypeData.ProcessingType
+
+import scala.collection.JavaConverters._
+import net.ceedubs.ficus.Ficus._
+
+trait ProcessCategoryService {
```

Comment:
Are we sure what we want to put here? Please remember, that category<->processType will probably change. Let's put some comment, so that folks won't put too much responsibilites here